Microsoft.Toolkit.Services/Services/Weibo/WeiboOAuthRequest.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Services.OAuth;
+using Newtonsoft.Json;
+
+namespace Microsoft.Toolkit.Services.Weibo
+{
+    /// <summary>
+    /// OAuth request.
+    /// </summary>
+    internal class WeiboOAuthRequest
+    {
+        private static HttpClient client;
```

Comment:
Can we make sure all private member variables start with "_"?

Reply:
Yes, it is a recommended rule. Will update it soon.